e2e/cli-e2e/tests/__snapshots__/compare.e2e.test.ts.snap

```diff
@@ -0,0 +1,705 @@
+        "title": "Disallow usage of the return value of ReactDOM.render",
+        "value": 0,
+      },
+      {
+        "displayValue": "passed",
+        "plugin": {
+          "slug": "eslint",
+          "title": "ESLint",
+        },
+        "score": 1,
+        "slug": "react-no-string-refs",
+        "title": "Disallow using string references",
+        "value": 0,
+      },
+      {
+        "displayValue": "passed",
+        "plugin": {
+          "slug": "eslint",
+          "title": "ESLint",
+        },
+        "score": 1,
+        "slug": "react-no-unescaped-entities",
+        "title": "Disallow unescaped HTML entities from appearing in markup",
+        "value": 0,
+      },
+      {
+        "displayValue": "passed",
+        "plugin": {
+          "slug": "eslint",
+          "title": "ESLint",
+        },
+        "score": 1,
+        "slug": "react-no-unknown-property",
+        "title": "Disallow usage of unknown DOM property",
+        "value": 0,
+      },
+      {
+        "displayValue": "passed",
+        "plugin": {
+          "slug": "eslint",
+          "title": "ESLint",
+        },
+        "score": 1,
+        "slug": "react-require-render-return",
+        "title": "Enforce ES5 or ES6 class for returning value in render function",
+        "value": 0,
+      },
+    ],
+  },
+  "categories": {
+    "added": [],
+    "changed": [
+      {
+        "scores": {
+          "after": 0.6242,
+          "before": 0.62461,
+          "diff": -0.00041000000000002146,
+        },
+        "slug": "code-coverage",
+        "title": "Code coverage",
+      },
+      {
+        "scores": {
+          "after": 0.7692307692307693,
+          "before": 0.5384615384615384,
+          "diff": 0.23076923076923084,
+        },
+        "slug": "code-style",
+        "title": "Code style",
+      },
+    ],
+    "removed": [],
+    "unchanged": [
+      {
+        "score": 0.6842105263157895,
+        "slug": "bug-prevention",
+        "title": "Bug prevention",
+      },
+    ],
+  },
+  "commits": Any<Object>,
+  "date": Any<String>,
+  "duration": Any<Number>,
+  "groups": {
+    "added": [],
+    "changed": [
+      {
+        "plugin": {
+          "slug": "coverage",
+          "title": "Code coverage",
+        },
+        "scores": {
+          "after": 0.6242,
+          "before": 0.62461,
+          "diff": -0.00041000000000002146,
+        },
+        "slug": "coverage",
+        "title": "Code coverage metrics",
+      },
+      {
+        "plugin": {
+          "slug": "eslint",
+          "title": "ESLint",
+        },
+        "scores": {
+          "after": 0.7142857142857143,
+          "before": 0.5,
+          "diff": 0.2142857142857143,
+        },
+        "slug": "suggestions",
+        "title": "Suggestions",
+      },
+    ],
+    "removed": [],
+    "unchanged": [
+      {
+        "plugin": {
+          "slug": "eslint",
+          "title": "ESLint",
+        },
+        "score": 0.9166666666666666,
```

Comment:
This actually goes for the `score` as well then. But OOS of course. WDYT?

Reply:
Answered here: https://github.com/code-pushup/cli/pull/556#discussion_r1521227569